### PR TITLE
add documentation for GET /system/health

### PR DESCRIPTION
**Pull request by cpanato**
Code PR: https://github.com/mattermost/platform/pull/6331

### CODE REVIEW

File: v4/source/system.yaml [hunk 1]
```diff
@@ -304,3 +304,22 @@
           $ref: '#/responses/Unauthorized'
         '501':
           $ref: '#/responses/NotImplemented'
+
+  '/system/health':
+    get:
+      tags:
+        - system
+      summary: Retrieve if the system are health or unhealthy
```
Comment (jwilander) [inline v4/source/system.yaml]: Can this be updated to: `Check system health`

File: v4/source/system.yaml [hunk 2]
```diff
@@ -304,3 +304,22 @@
...
+      description: |
+        Retrieve if the system are health or unhealthy based on the configuration `GoRoutineHealthThreshold`.
+        ##### Permissions
```
Comment (jwilander) [inline v4/source/system.yaml]: And this to:

```
Check if the system is health or unhealthy based on the configuration setting `GoRoutineHealthThreshold`.

__Minimum server version__: 3.10
```

(Need to add minimum server version for everything past 3.9)

File: v4/source/system.yaml (hunk 1 above)
Comment (cpanato) [inline v4/source/system.yaml]: done

File: v4/source/system.yaml (hunk 2 above)
Comment (cpanato) [inline v4/source/system.yaml]: done

**Pull request merged**